teamsapp.yml

```diff
@@ -93,3 +93,4 @@ deploy:
       # You can replace it with your existing Azure Resource id
       # or add it to your environment variable file.
       resourceId: ${{BOT_AZURE_APP_SERVICE_RESOURCE_ID}}
+projectId: 9ae39cb4-2b24-460f-ae64-f325f90f961a
```

Comment:
minor: please remove the projectId field when creating template based on this project.